adding textarea element example

chrisdavidmills opened the pull request:
As per https://github.com/mdn/interactive-examples/issues/754

wbamberg commented on live-examples/html-examples/forms/textarea.html:
```diff
@@ -0,0 +1,40 @@
+        <label for="min-max">minlength/maxlength example:</label>
+        <textarea id="min-max" name="min-max"
+                  rows="4" cols="50"
+                  minlength="3" maxlength="45">
```
minlength did not work for me here (Chrome 66, Firefox 61). At least, it let me delete everything in the box. Is that the intended behavior?

chrisdavidmills commented:
thanks for the feedback @wbamberg! I have simplified it quite a bit, getting rid of all the examples except the basic and sized ones (which I renamed to advanced).

Other points to note:

* I added a maxlength constraint to the advanced one, to show how that works
* I reduced the height of it so it all fits into the output - no scrollbars. This has the added side effect that it makes the textarea itself scroll, and I think it is useful to see how that looks.
* I've removed minlength altogether — the trouble with it is that it behaves differently to maxlength — it doesn't stop people reducing the number of characters below the specified minlength, but the value is invalid if that is the case. So to demonstrate this would require an extra set of complication, setting up valid/invalid styles, etc. And I'm not sure if it worth it really. It would make things additionally complicated.

chrisdavidmills commented:
OK, makes sense - column size reduced!
